.github/workflows/build.yml

```
@@ -13,7 +13,7 @@ jobs:
 
       - name: Build and publish Docker image
         if: github.ref == 'refs/heads/main'
-        uses: elgohr/Publish-Docker-Github-Action@33a481be3e179353cb7793a92b57cf9a6c985860 # v4
+        uses: elgohr/Publish-Docker-Github-Action@v5 # v4
```

A very much delayed thanks! But we rather pin the dependency versions by hash for security reasons
```suggestion
        uses: elgohr/Publish-Docker-Github-Action@eb53b3ec07136a6ebaed78d8135806da64f7c7e2 # v5
```